src/tools/Reporting/Reporting/Build.cs

```diff
@@ -25,5 +25,12 @@ public sealed class Build
         public string BuildName { get; set; }
 
         public DateTime TimeStamp { get; set; }
+
+        public Dictionary<string, string> AdditionalData { get; set; } = new Dictionary<string, string>();
+
+        public void AddData(string key, string payload)
+        {
+            AdditionalData.Add(key, payload);
```

Comment:
Should this have an add/update/etc pattern? What happens if I need to change some set of data?

May be better to just let the callsite manipulate the dictionary.

Reply:
FYI it's following the format of ```Test.cs``` https://github.com/dotnet/performance/blob/602872a6c8da1369c3d522676a50f053af3da4ac/src/tools/Reporting/Reporting/Test.cs#L43-L46
Unless we have a centralized place for methods manipulating the dictionary somewhere, I'd prefer callsite just accessing the dictionary, otherwise we might end up duplicating these methods for other library classes.  @DrewScoggins

Reply:
I did not put much thought into accessing this when I first added this in Test.cs. I think just moving to a place where we manipulate the dictionary directly is perfectly fine here.